internal/vul/table.go

```diff
@@ -100,28 +102,43 @@ func (t *table) addRow(r Row) {
 	t.Rows = append(t.Rows, r)
 }
 
-func (t *table) dump(w io.Writer) {
+func (t *table) dump(w io.Writer) error {
```

Comment:
not positive if this is an interface definition somewhere - if so would appreciate feedback for expectations on linting.  (IE should we flag as `//nolint:errcheckl`?)